tests/test_importexport.py

```diff
@@ -0,0 +1,110 @@
+import os
+
+from PyQt5 import QtCore
+from PyQt5.QtWidgets import QFileDialog, QDialogButtonBox, QMessageBox
+
+from vorta.models import BackupProfileModel, SourceFileModel
+from vorta.views.import_window import ImportWindow
+
+
+def test_import_success(qapp, qtbot, rootdir, monkeypatch):
+    # Do not change this file, as it also tests restoring from older schema versions
+    GOOD_FILE = os.path.join(rootdir, 'profile_exports', 'valid.json')
+
+    def getOpenFileName(*args, **kwargs):
+        return [GOOD_FILE]
+
+    monkeypatch.setattr(
+        QFileDialog, "getOpenFileName", getOpenFileName
+    )
+
+    main = qapp.main_window
+    main.profile_import_action()
+    import_dialog: ImportWindow = main.window
+    import_dialog.overwriteExistingSettings.setChecked(True)
+
+    qtbot.mouseClick(import_dialog.buttonBox.button(QDialogButtonBox.Ok), QtCore.Qt.LeftButton)
+    qtbot.waitUntil(lambda: qapp.activeWindow() is None)
```

Comment:
This fixes the failing test on macOS. We just wait for the signal instead of the window.

```
import pytest
...
qtbot.waitSignal(import_dialog.profile_imported, **pytest._wait_defaults)
```

This waits for your import signal with a longer timeout, since some CI platforms can be a bit slow at times.